mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -274,51 +324,136 @@ public override void Run (IEnumerable<string> args)
 				"Do not delete members that don't exist in the assembly, but rather mark them as preserved.",
 				v => PreserveTag = "true" },
 		};
-		var assemblies = Parse (p, args, "update", 
+		var assemblyNames = Parse (p, args, "update", 
 				"[OPTIONS]+ ASSEMBLIES",
 				"Create or update documentation from ASSEMBLIES.");
-		if (assemblies == null)
+		if (assemblyNames == null)
 			return;
-		if (assemblies.Count == 0)
+		if (assemblyNames.Count == 0)
 			Error ("No assemblies specified.");
 
-		foreach (var dir in assemblies
-				.Where (a => a.Contains (Path.DirectorySeparatorChar))
-				.Select (a => Path.GetDirectoryName (a)))
-			assemblyResolver.AddSearchDirectory (dir);
+		ProcessAssemblyOwners (ref assemblyNames);
+
+		// check to see if there are two dropns entries
```

Comment:
Why do we care? What's so special about there being 1 vs. 2 "droppedAssemblies" entries? What happens if there's 3? 4?